src/handlers/notification/index.js

```diff
@@ -187,6 +188,8 @@ const processMessage = async (msg, span) => {
     const payloadForCallback = decodedPayload.body.toString()
     let callbackHeaders
 
+    let jwsSigner = getJWSSigner(from)
```

Comment:
This seems unecessary...as we would only create a JWSSigner when the message is sent from the Switch (which is done below).

i.e. why wouldn't all the messages here be signed by the switch? :D

For example why do we not sign the message on https://github.com/mojaloop/bulk-api-adapter/pull/90/files#diff-2ad92add22f92879b719bfa1dba689027c5282e313802338863d88571f0293c1R227 or any of the other alternative scenarios here?

Reply:
I added the jws signing for all the messages as we discussed. @elnyry-sam-k please review this.